### PR TITLE
Update packages.json with libnx

### PR DESCRIPTION
**Pull request by jyapayne**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -387,6 +387,20 @@
     "license": "BSD",
     "web": "https://github.com/nim-lang/nimble"
   },
+  {
+    "name": "libnx",
+    "url": "https://github.com/jyapayne/nim-libnx",
+    "method": "git",
+    "tags": [
+      "switch",
+      "nintendo switch",
```
Comment (Yardanico) [inline packages.json]: no spaces in tags please

Comment (Yardanico) [reply packages.json]: https://github.com/nim-lang/packages/pull/750#discussion_r189970083

**Comment by Yardanico** [comment]
Ping @notTito - "E: morpheus: unable to fetch repository https://github.com/notTito/morpheus 404 Not Found"

**Comment by jyapayne** [comment]
@Yardanico shouldn't the package scanner just ignore invalid or removed repos?

**Comment by Yardanico** [comment]
It shouldn't

**Comment by jyapayne** [comment]
Hey @dom96, is there a way to keep this structure? Like I want to have these wrappers still available to users but also write a Nim friendly wrapper in src/libnx. Is that possible?

**Pull request merged**